ng-dev/release/publish/actions/cut-stable.ts

```diff
@@ -29,9 +29,11 @@ export class CutStableAction extends ReleaseAction {
     const {branchName} = this.active.releaseCandidate!;
     const newVersion = this._newVersion;
     const isNewMajor = this.active.releaseCandidate?.isMajor;
+    const compareVersionForReleaseNotes = this.active.releaseCandidate!.version;
```

Comment:
yeah, good question. In the current way, the changelog will remain incremental.

 One downside with re-building the whole changelog from the last patch is that the previous next pre-releases would still have entries in the changelog (and there would be duplication)

Reply:
I think that is actually what we want correct?  We want to have a changelog entry that shows all commits from `x.0.0-next.0` to `x.0.0` even though we have entries in the changelog for `-next.0`, `-next.1`, `-rc.0`, etc.

Essentially our cut to stable changelog entry should be made up of primarily duplicates, but it provides information for anyone who is migrating from the old stable to the new stable, the changes that have occured?

Reply:
Yeah, that sounds reasonable. I'll try making that change.

I brought this up because I remember in FW that some caretaker also looked into deleting the old individual next entries when he compiled the large entry for the stable. Although I could have misunderstood/or this was by accident.

Reply:
In an ideal world we probably would do that, but we can call that out of scope for now.

Reply:
sounds good to me